policyengine_us/tests/policy/contrib/crfb/tax_employer_all_payroll_tax.yaml

```diff
@@ -2,14 +2,16 @@
   period: 2026
   absolute_error_margin: 0.01
   # Refer to test object, not just the file
-  reforms: policyengine_us.reforms.crfb.tax_employer_payroll_tax.tax_employer_payroll_tax_reform_object
+  reforms: 
+    - policyengine_us.reforms.crfb.tax_employer_medicare_tax.tax_employer_medicare_tax_reform_object
+    - policyengine_us.reforms.crfb.tax_employer_social_security_tax.tax_employer_social_security_tax_reform_object
   input:
-    gov.contrib.crfb.tax_employer_payroll_tax.in_effect: true
+    gov.contrib.crfb.tax_employer_medicare_tax.in_effect: true
```

Comment:
i think we want the social security parameter active here too, as that'll reflect how it works in the app. @PavelMakarchuk could you advise on tests that activate multiple structural reform parameters